book/working_with_strings.md

```diff
@@ -146,7 +179,7 @@ For example, let's say we have a variable called `$name` and we want to greet th
 > let name = "Alice"
 > $"greetings, ($name)"
 greetings, Alice
-```
+````
```

Comment:
this looks incorrect, right?

Reply:
done in https://github.com/nushell/nushell.github.io/pull/1083/commits/32067f1dd7f61b1f008e54b401f2991ff9641f76